hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AbfsConfiguration.java

```diff
@@ -1240,8 +1244,8 @@ public boolean getRenameResilience() {
     return renameResilience;
   }
 
-  void setRenameResilience(boolean actualResilience) {
```

Comment:
any reason to cut this? I presume it means no tests are using it...

Reply:
Yes this code was not used any where so removed it